dirac.cfg

```diff
@@ -57,6 +57,9 @@ Registry
       # Real VOMS VO name, if this VO is associated with VOMS VO
       VOMSName = lhcb
 
+      # Registered identity provider associated with VO
+      IdP = ChechIn
```

Comment:
```suggestion
      IdP = CheckIn
```
Where is this read?

Reply:
`AuthHandler` read it through `Registry.getIdPForGroup`